idl/github.com/uber/cadence/history.thrift

```diff
@@ -208,6 +219,23 @@ service HistoryService {
       4: ShardOwnershipLostError shardOwnershipLostError,
     )
 
+  /**
+  * Reset the volatile information in mutable state of a given workflow.
+  * Volatile information are the information related to client, such as:
+  * 1. StickyTaskList
+  * 2. StickyScheduleToStartTimeout
+  * 3. ClientLibraryVersion
+  * 4. ClientFeatureVersion
+  * 5. ClientImpl
+  **/
+  ResetMutableStateResponse ResetMutableState(1: ResetMutableStateRequest resetRequest)
```

Comment:
ResetMutableState sounds like it is going to reset the mutable state. It might be better to just call it ResetStickyTaskList.

Reply:
sure

Reply:
changed to ResetMutableStateStickyness

let me know if this name works